server/schemas/schemaController.js

```diff
@@ -79,6 +79,20 @@ module.exports = {
     schemaModel.createTemplate(request, response, templateSuccessOrFailCallback);
   },
 
+  fetchTemplate: function(request, response) {
```

Comment:
follow local style – add documentation and group with the READ methods. controller code needs to maximize readability above all else.